native/acter/src/api/message.rs

```diff
@@ -291,10 +295,207 @@ impl RoomEventItem {
                     }
                 }
             }
+
             TimelineItemContent::OtherState(s) => {
                 info!("Edit event applies to a state event");
-                me.event_type(s.content().event_type().to_string());
+                let event_type = s.content().event_type().to_string();
+                me.event_type(event_type.clone());
+                if let AnyOtherFullStateEventContent::RoomName(c) = s.content() {
+                    if let FullStateEventContent::Original {
+                        content,
+                        prev_content,
+                    } = c
+                    {
+                        let msg_content = match (&content.name, &prev_content) {
+                            (new, Some(prev)) => match &prev.name {
+                                Some(old) => MsgContent::from_text(format!("{old} -> {new}")),
```

Comment:
fine for now, but in the future we probably want to leave that to the UI to decide as on LTR this is probably incorrect notation ;) .